src/lib/prompts-helper.ts

```diff
@@ -0,0 +1,9 @@
+import chalk from 'chalk';
+
+const onCancel = (text: string): void => {
+  console.log(chalk.red(text));
```

Comment:
I don't think this text needs to be red. Also drop `-helper` from the name of this file.